tests/test_search.py

```diff
@@ -204,6 +205,7 @@ def test_different_similarity(self):
         self.assertAlmostEqual(hits[9].score, 4.33320, places=5)
 
     def test_rm3(self):
+        self.searcher = LuceneSearcher(f'{self.index_dir}lucene-index.cacm')
```

Comment:
I noticed that the tests worked fine when I stopped this test from running:

https://github.com/castorini/pyserini/blob/962ba799077ab8eea9a77fed9746965087432869/tests/test_search.py#L161

Since we are doing the setup once for all methods, it looks to me like something in that function is altering the searcher in such a way that it caused `test_rm3` to fail. To fix this, I decided to reinstantiate the lucenesearcher instance for `test_rm3` and the tests passed.